File: Selenium/Project/Activity2.java
package JOBBOARD;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity2 {
    WebDriver driver;

    @BeforeClass
    public void Beforeclass() {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs");

   }
    @Test
    public void HeadLine() throws InterruptedException {
       Thread.sleep(1200);
       String headline= driver.findElement(By.className("entry-title")).getText();
        System.out.println("The Title Is " + headline);
        Assert.assertEquals("Welcome to Alchemy Jobs",headline);
        String head="Welcome to Alchemy Jobs";
        if (headline == head){
        driver.close(); }

    }
}

File: Selenium/Project/Activity2.py
# Import webdriver from selenium
import pytest
import time

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())
# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Open the browser to the URL
    driver.get("https://alchemy.hguy.co/jobs")
    print("Page Title is ", driver.title)
    time.sleep(8)

    head = driver.find_element(By.CLASS_NAME ,"entry-title")
    headline =head.text
    print("The Title Is " , headline)
    head ="Welcome to Alchemy Jobs"
    if headline==head:
      driver.close()




